pages/1.12/release-notes/1.12.3/index.md

```diff
@@ -0,0 +1,150 @@
+
+DC/OS 1.12.3 includes the following components:
+- Apache Mesos 1.7.x [change log](https://github.com/apache/mesos/blob/4f21147a9334ef9ed84dcf11742ce448062f3ec/CHANGELOG).
+- Marathon 1.7.203 [change log](https://github.com/mesosphere/marathon/blob/48bfd6000c544df5ae03de04b42b019d5e9dbd4b/changelog.md).
+- Metronome 0.5.71 [change log](https://github.com/dcos/metronome/blob/391637cc19cd6136e8733ff8b684aed31b2cf672/changelog.md).
+
+<!-- <p class="message--note"><strong>NOTE: </strong>DC/OS 1.12.1 release supports new CoreOS and Docker versions as listed in the <a href="../../../version-policy">compatibility matrix</a>.</p> -->
+
+# Release Summary
+
+DC/OS is a distributed operating system that enables you to manage resources, application deployment, data services, networking, and security in an on-premise, cloud, or hybrid cluster environment. 
+
+# Issues Fixed in DC/OS 1.12.3
+The issues that have been fixed in DC/OS 1.12.3 are grouped by feature, functional area, or component. Most change descriptions include one or more issue tracking identifiers for reference.
+
+## Command-Line Interface (CLI)
+- DCOS-42928 - This release includes `docker ps` output in `dcos-diagnostic` bundle.
+
+## GUI
+- DCOS-45863 - Currently, the `Settings/LDAP Directory/Add Directory/Authentication` dialog has a mandatory `lookup-dn` and an optional `lookup-password`. This release provides a possibility to select between `Anonymous Bind` and `LDAP Credentials` that translates to the following:
+    - If `Anonymous Bind` is selected, the `JSON` sent to Bouncer `/ldap/config` API has no `lookup-dn` or `lookup-password` field.
+    - If `LDAP Credentials` is selected, use the current behavior. The `JSON` sent to Bouncer `/ldap/config` API has `lookup-dn` and `lookup-password` field. The `lookup-password` field may be an empty string.
+
+## Installation
+ - COPS-4282, DCOS_OSS-4613 - If you run the `dcos_generate_config` command with the `-validate` option, the command validates the configuration settings in your `config.yaml` file. In some cases, this option issued warning messages that validation failed for parameters that are no longer used. For example, some SSH parameters, such as `ssh_key_path` and `ssh_user`, have been deprecated. Previously, if you ran `dcos_generate_config` with the `-validate` option to check your configuration settings and these parameters were not specified, the command reported that the validation of configuration parameters had failed. With this release, the `--validate` option does not return validation failure messages for parameters that are no longer required for installation.
+
+- DCOS-15890 - The Preflight check on advance installer shows misleading information. This release improves an error message in case Docker is not running at the start of installation.
+
+## Marathon
+- COPS-3554 - The watcher loop process monitors and re-registers (if necessary) the Marathon leader after re-election.
```

Comment:
A bit more detail would be appreciated, how about: "Fixed a rare problem where a follower Marathon might try to proxy to a non-leader instance.  The fix adds a watcher loop process that monitors and re-registers (if necessary) the Marathon leader after re-election."